code/game/objects/items/stacks/medical.dm

```diff
@@ -274,6 +274,19 @@
 	splittable = FALSE	// Is the stack capable of being splitted?
 	preloaded_reagents = list("silicon" = 4, "ethanol" = 10, "lithium" = 4)
 
+/obj/item/stack/medical/advanced/bruise_pack/mending_ichor
+	name = "mending ichor"
+	singular_name = "mending ichor"
+	desc = "An ichor that can be used to mend physical trauma."
+	icon_state = "mending_ichor"
+	heal_brute = 8
+	origin_tech = list(TECH_BIO = 2)
```

Comment:
Already defined in parent type. That means your mending_ichor type will already inherits these values from the bruise_pack parent, so you don't have to define them.